capstone-backend/app.js

```diff
@@ -631,6 +663,66 @@ function requestToken(res, redirect_uri, code, userInfo, params) {
         });
 }
 
+function requestSpotifyToken(res, redirect_uri, code, params) {
```

Comment:
I know we talked about you having a future diff that will update the functions to => , but I would recommend the new methods you have to use it already, so that less refactoring later on.